src/components/Daily.js

```diff
@@ -23,14 +23,14 @@ export default class Daily extends Component {
         </div>
         <div className="daily-itemlistwrapprt">
           <ul className="daily-itemlist">
-          { _.map(dailyTop, (a) => {
+          { _.map(dailyTop, (a, idx) => {
             const pubDate = date2yyyymmdd(a.publishedDate, '.')
             if (a.state == 'published') {
               let imageSet = imageComposer(a)
               let thumbnail = imageSet.mobileImage
               let url = '/a/' + a.slug
               return (
-                <li className="daily-item" key={a._id}>
+                <li className="daily-item" key={idx}>
```

Comment:
1.
Although `{a.id || idx}` is also a way, 
I first prefer that decoupling Frontend logic from Backend value when they are not related.

2.
Secondly, I prefer `{a.id}` because it make sure all DOM use value from Backend.

3.
I don't prefer use `{a.id || idx}`:
For the value of key, this way might be ambiguity.

PS:
Sometimes, we need share the same value between Frontend & Backend parts together, 
but here we are not.